app/segments/drill/[id]/submission/input.js

```diff
@@ -243,19 +244,27 @@ function createOutputData(drillInfo, inputValues, attemptShots, uid, did) {
           shot.carry = inputValues[j].carry;
           break;
 
+        case "strokesTaken":
+          shot.strokes = inputValues[j].strokes;
+          break;
+
+        case "break":
+          shot.break = attemptShots[j].target[1];
+          break;
+
         case "sideLanding":
           shot.sideLanding = Number(inputValues[j].sideLanding);
           sideLandingTotal += Number(inputValues[j].sideLanding);
           break;
 
         case "proxHole":
           shot.proxHole = calculateProxHole(
-            attemptShots[j].target,
+            attemptShots[j].target[0],
```

Comment:
ye I'm not really vibing with this whole target[0] and target[1] business. Change this to an object or sth so the code is readable